docs/src/main/sphinx/connector/hive.rst

```diff
@@ -319,6 +319,58 @@ Hive connector documentation.
       - Maximum number of partitions to drop in a single query.
       - 100,000
 
+Hive native reader and writer configuration
```

Comment:
I dont think this deserves a separate section .. we should work it into the file format sections . Lets sync up with @m57lyra